examples/ExampleStyled/index.html

```diff
@@ -16,7 +16,7 @@
     <section id="textOutput-content"></section>
     <section id="gridOutput-content"></section>
     <section id="soundOutput-content"></section>
-    <script src="https://cdn.rawgit.com/processing/p5.accessibility/298ccdf6/dist/p5-accessibility.js"></script>
```

Comment:
In the examples, could we link to the library on cdn instead of a local file?

Reply:
also after we get through all the PRs we should get a new CDN. I'm working on #20 this weekend and doing some curriculum experiments. ;D

Reply:
Sounds good. Yes, will revert to the old CDN for now.
In that case, shall I add semantic versioning after #20 ?